.changeset/long-icons-warn.md

```diff
@@ -0,0 +1,5 @@
+---
+"@astrojs/cli-kit": patch
```

Comment:
I would say this should _at least_ be a `minor`?

Reply:
```suggestion
"@astrojs/cli-kit": minor
```